src/modules/esl-media/core/esl-media.ts

```diff
@@ -75,6 +75,8 @@ export class ESLMedia extends ESLBaseElement {
   @attr() public loadClsAccepted: string;
   /** Class / classes to add when media is declined */
   @attr() public loadClsDeclined: string;
+  /** load-condition . Default: `all` */
```

Comment:
extra cosmetic (smt like that):
```suggestion
  /** Media loading conditions. Default: `all` */
```

Reply:
Fixed in #778